wapitiCore/attack/mod_check_cookies_headers.py

```diff
@@ -0,0 +1,43 @@
+import requests
+from wapitiCore.attack.attack import Attack
+from wapitiCore.net.web import Request
+from wapitiCore.language.vulnerability import Additional
+
+class mod_check_cookies_headers(Attack):
+
+    name = "check_cookies_headers"
+
+    def check_secure_flag(self, cookie: object):
+
+        return cookie.secure
+
+    def check_httponly_flag(self, cookie: object):
+
+        return bool('HttpOnly' in cookie._rest)
```

Comment:
"in" statement already returns a boolean, bool is useless

Reply:
Fixed